packages/client-config/src/client-config-contributor/client_config_contributor_factory.ts

```diff
@@ -58,12 +68,12 @@ export class ClientConfigContributorFactory {
         new Custom1_1(),
       ],
 
-      // Legacy config is derived from V1.2 (latest) of unified default config
+      // Legacy config is derived from V1.3 (latest) of unified default config
       [ClientConfigVersionOption.V0]: [
         new Auth1_1(),
```

Comment:
Do we have `groups` in Gen1 config?

Reply:
I believe for the Gen1 config it is `userPoolGroups`

Reply:
Then should we add that conversion for legacy config as well?

Reply:
Sorry looked at the wrong file, we don't have groups in Gen1 config.